server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestAddVotingConfigExclusionAction.java

```diff
@@ -47,15 +47,18 @@ public String getName() {
 
     @Override
     protected RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
-        String nodeName = request.param("node_name");
-        AddVotingConfigExclusionsRequest votingConfigExclusionsRequest = new AddVotingConfigExclusionsRequest(
-            new String[]{nodeName},
-            TimeValue.parseTimeValue(request.param("timeout"), DEFAULT_TIMEOUT, getClass().getSimpleName() + ".timeout")
-        );
         return channel -> client.execute(
             AddVotingConfigExclusionsAction.INSTANCE,
-            votingConfigExclusionsRequest,
+            resolveVotingConfigExclusionsRequest(request),
```

Comment:
Tests on the PR are failing because this is no lazy-resolved. Please move the request creation back to the prepareRequest method. Thank you.